.github/workflows/pull-requests.yaml

```
@@ -126,6 +126,15 @@ jobs:
     if: ${{ always() && (needs.build.result == 'success' || needs.resolve_assets.result == 'success') }}
 
     steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Prepare workspace
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:13: Double quote to prevent globbing and word splitting

(shellcheck)